docker/db/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM mariadb:10.1
```

Comment:
Likewise this should probably be a shared image that has the base tables, but this looks good for now (I haven't published anything with the base tables yet anyway)